policy-engine/src/main.rs

```diff
@@ -56,10 +57,12 @@ fn main() -> Result<(), Error> {
     let state = graph::State {
         upstream: opts.upstream,
     };
+
+    let graph_path = opts.path_namespace.clone() + "/v1/graph";
```

Comment:
If I'm reading the ticket correctly, this will result in `/api/updates_mgmt/v1/v1/graph` for both services. What's the semantic difference between the two `v1`? Is it fine that they are duplicated?

Reply:
I *think* they would use `api/updates_mgmt` as the namespace but I don't know exactly. Anyway from Cincinnati perspective the namespace is opaque and we shouldn't care about the content.

Reply:
I see. I was just confused about version prefixes semantics.